Procfile.dev

```diff
@@ -1,3 +1,3 @@
-web: bundle exec puma -C config/puma.rb
-stream: yarn run start
-webpack: ./bin/webpack-dev-server
+web: PORT=3000 bundle exec puma -C config/puma.rb
```

Comment:
I think this already honored `PORT` if set, and I'd rather not hardcode it to something...